tests/js/core/widget/core/SectionChanger/SectionChanger.js

```diff
@@ -25,8 +25,92 @@
 		});
 
 
-// unit test, one case
-		QUnit.test("handleEvent", function test(assert) {
+		QUnit.test("_configure", function (assert) {
+			var sectionChanger = new SectionChanger();
+
+			sectionChanger._configure();
+
+			expect(11);
+
+			assert.equal(sectionChanger.options.items, "section", "items is 'section'");
+			assert.equal(sectionChanger.options.activeClass, "ui-section-active", "activeClass is 'ui-section-active'");
+			assert.equal(sectionChanger.options.circular, false, "circular is false");
+			assert.equal(sectionChanger.options.animate, true, "animate is true");
+			assert.equal(sectionChanger.options.animateDuration, 100, "itanimateDurationms is 100");
+			assert.equal(sectionChanger.options.orientation, "horizontal", "orientation is 'horizontal'");
+			assert.equal(sectionChanger.options.changeThreshold, -1, "changeThreshold is -1");
+			assert.equal(sectionChanger.options.useTab, false, "useTab is 'false'");
+			assert.equal(sectionChanger.options.fillContent, true, "fillContent is 'true'");
+			assert.equal(sectionChanger.options.model, null, "model is null");
+			assert.equal(sectionChanger.options.directives, null, "directives is null");
+		});
+
+		QUnit.test("_build", function (assert) {
+			var element = document.getElementById("emptysectionchanger"),
+				sectionChanger = new SectionChanger();
+
+			sectionChanger.options.orientation = "horizontal";
+			sectionChanger._build(element);
+
+			expect(3);
+
+			assert.equal(sectionChanger.activeIndex, 0, "activeIndex is '0'");
+			assert.equal(sectionChanger.beforeIndex, 0, "beforeIndex is 0");
+			assert.equal(sectionChanger.orientation, "horizontal", "orientation is horizontal");
+		});
+
+		QUnit.test("_init", function (assert) {
+			var element = document.getElementById("emptysectionchanger"),
+				sectionChanger = new SectionChanger();
+
+			sectionChanger.element = element;
+			sectionChanger._build(element);
+			sectionChanger._init(element);
+
+			expect(2);
+
+			assert.equal(sectionChanger.activeIndex, 0, "activeIndex is horizontal");
```

Comment:
The `_init` method is much more complicated. In my opinion this test actually not testing whole `_init' method. You should remove this test.

Reply:
I know what you're talking about. I will remove it.